css/subject-viewer.styl

```diff
@@ -51,6 +51,7 @@
     .text-viewer
       background-color: #404040
       color: white
+      min-width: 30em
```

Comment:
No overlap here, I'm afraid. Could this be caused by viewport width? In that case the sizing should maybe use `vw` instead of `em`.
![Screenshot showing the text viewer side-by-side with the task area in Chrome](https://cloud.githubusercontent.com/assets/59547/24873379/7f8f2458-1e18-11e7-8e40-2cb808af2d04.png)